src/position.cpp

```diff
@@ -1156,12 +1157,33 @@ bool Position::see_ge(Move m, Value threshold) const {
       return true;
 
   bool opponentToMove = true;
-  occupied = pieces() ^ from ^ to;
+  occupied = pieces() ^ to;
```

Comment:
I would prefer to keep it as it is and check for the opposite condition below, since `NO_GATE_TYPE` is the default case.